configure

```diff
@@ -13,16 +17,18 @@ del _
 
 import sys
 from distutils.spawn import find_executable as which
-if sys.version_info[:2] != (2, 7):
-  sys.stderr.write('Please use Python 2.7')
-
-  python2 = which('python2') or which('python2.7')
-
-  if python2:
-    sys.stderr.write(':\n\n')
-    sys.stderr.write('  ' + python2 + ' ' + ' '.join(sys.argv))
 
+current_python = 'python{0}.{1}'.format(*sys.version_info)
+valid_pythons = ('python3.7', 'python3.6', 'python3.5', 'python2.7')
+if current_python in valid_pythons:
+  import configure
+else:
+  sys.stderr.write('Please use ' + ', '.join(valid_pythons))
```

Comment:
Perhaps the error message can be improved a little? E.g.:
```py
sys.stderr.write(
    'Need one of {}, found {}'.format(','.join(valid_pythons), current_python))
```

Reply:
I pushed this.